sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt

```diff
@@ -83,6 +82,15 @@ class SettingsFragment : KinEcosystemBaseFragment<ISettingsPresenter, ISettingsV
         }
     }
 
+    override fun showTransferItem(show: Boolean) {
+        if(show) {
+            transferItem.visibility = VISIBLE
+        }
+        else {
+            transferItem.visibility = GONE
+        }
+    }
```

Comment:
can be simplified to one line:
`transferItem.visibility = if (show) VISIBLE else GONE`